src/04-modality-specific-files/05-task-events.md

```diff
@@ -110,16 +111,19 @@ sub-control01/
-| StimulusPresentationCode     | RECOMMENDED. URL to code of that was used to present the stimuli.                                                                                                                                                                |
+| Column name                      | Description                                                                                                                                                                                                                                                                                                                                   |
+| -------------------------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| StimulusPresentationOS           | RECOMMENDED. Operating system used to run the stimuli presentation software. Should mention the OS type (Windows, MacOS, Linux), distribution (if applicable: Ubuntu, Debian) and the version number or equivalent code name (Windows - 10; MacOS - Catalina or MacOS - 10.15; Linux - Ubuntu - Bionic Beaver or Linux - Ubuntu -18.04.5 LTS) |
+| StimulusPresentationSoftwareName | RECOMMENDED. Name of the software that was used to present the stimuli                                                                                                                                                                                                                                                                        |
+| StimulusPresentationSoftwareRRID | RECOMMENDED. Name ideally with its [Research Resource Identifier](https://scicrunch.org/resources) of the software that was used to present the stimuli Examples: the RRID for Psychtoolbox is 'SCR_002881' that of PsychoPy is 'SCR_006571'                                                                                                  |
+| StimulusPresentationVersion      | RECOMMENDED. Version of the software that was used to present the stimuli.                                                                                                                                                                                                                                                                    |
+| StimulusPresentationCode         | RECOMMENDED. DOI of the archive (for example on Zenodo, figshare...) for the code that was used to present the stimuli. Alternatively this could be a URL to code on Github, Gitlab...                                                                                                                                                        |
```

Comment:
```suggestion
| StimulusPresentationCode         | RECOMMENDED. DOI of the archive (for example on Zenodo, figshare...) for the code that was used to present the stimuli. Alternatively this could be a URL to code on GitHub, GitLab, etc.                                                                                                                                                     |
```